### ISSUE TITLE
.pro domain lookups don't work

### ISSUE THREAD

**Issue by twbryan** [richardpenman/whois#198]
```In [1]: import whois

In [2]: whois.whois("tech.pro")
Error trying to connect to socket: closing socket - [Errno 8] nodename nor servname provided, or not known
Out[2]:
{'domain_name': None,
 'registrar': None,
 'whois_server': None,
 'referral_url': None,
 'updated_date': None,
 'creation_date': None,
 'expiration_date': None,
 'name_servers': None,
 'status': None,
 'emails': None,
 'dnssec': None,
 'name': None,
 'org': None,
 'address': None,
 'city': None,
 'state': None,
 'registrant_postal_code': None,
 'country': None}```

**Comment by case-fastly** [comment]
According to https://www.iana.org/domains/root/db/pro.html, the whois server URL is now `whois.nic.pro` -- maybe that's the issue?

**Comment by skibine** [comment]
> According to https://www.iana.org/domains/root/db/pro.html, the whois server URL is now `whois.nic.pro` -- maybe that's the issue?

Nope. It is not the issue. And .pro domains - are not the only. Try .sc, for example maven.sc or nic.sc. But by the way, from console i can whois .pro domains (from python - nope), and .sc domains do not work either under the console or under Python. 
And _whois maven.sc --verbose_ shows "Using server whois2.afilias-grs.net" - it is correct, according to iana, and from web-interface "whois2.afilias-grs.net" - all works well. I don't know where to dig. It seems to me that there are several reasons and they came together at the same time.

**Comment by case-fastly** [comment]
Fwiw, these zones (`.pro`, `.sc`, etc) moved from the Afilias registry backend operator to Identity Digital on 2023-08-01. You might consider contacting the Technical Contact email address here -- https://www.iana.org/domains/root/db/pro.html -- and asking why `whois.nic.pro` doesn't seem to be working.

https://www.prnewswire.com/news-releases/donuts-inc-and-afilias-inc-rebrand-to-identity-digital-301572401.html

https://en.wikipedia.org/wiki/Afilias

**Comment by BillWeissABK** [comment]
For what it's worth, this is working without any change for me. I think @mzpqnxow 's comment is correct, this doesn't need special whois servers anymore.

**Comment by lelutin** [comment]
I can confirm what @BillWeissABK said. with python 3.11.9 the same excerpt gives me a bunch of information without errors.

@twbryan can you try again on the latest commit on the master branch to see if the issue is fixed for you too?

**Comment by bite-parry** [comment]
.io domain lookups also don't work.

**Comment by ParryQiu** [comment]
.io domain lookups also don't work.

**Comment by mzpqnxow** [comment]
> .io domain lookups also don't work.

Can you enter a separate issue for that?
